sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -126,14 +141,15 @@ def test_pipeline_transformers_workflow(self):
     from .testdata.pipeline_transformers import transform_pipeline
     self._test_pipeline_workflow(transform_pipeline, 'pipeline_transformers.yaml')
 
-  def _test_pipeline_workflow(self, pipeline_function, pipeline_yaml, generate_pipelinerun=False):
+  def _test_pipeline_workflow(self, pipeline_function, pipeline_yaml, generate_pipelinerun=False, normalize_compiler_output=None):
```

Comment:
hmm, maybe we should call this `normalize_compiler_output_function` to make it clear what this parameter is
and break it onto a new line if line exceeds 120 characters

Reply:
Updated, Thanks